config/environments/staging.rb

```diff
@@ -37,7 +37,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  # config.active_storage.service = :local # TODO enable when rails is upgraded
```

Comment:
Do we even need to enable that? Are we handling any files? Is active storage needed for anything else?

Reply:
I think we'll be taking csv uploads, so will need